### DISCUSSION TITLE
GUI based React-Table code generator

### DISCUSSION THREAD

**Discussion by akamittal** [TanStack/table#2993] [Show and tell]
Hi! This is Akash Mittal.

I have created a GUI based code generator which can give you whole code snippet. Currently it supports features like -
1. Headers
2. Footers
3. Pagination
4. Sorting
5. Global Search
6. Filters

I will keep it updating to support all the features of react-table. Hope this helps everybody.
Here is the link - [https://www.akashmittal.com/gui-utility-to-generate-react-table-code/](https://www.akashmittal.com/gui-utility-to-generate-react-table-code/)

**Comment by ravisojitra** [comment]
@akamittal Please can you provide source code of it? or github repo?
